gradle/mockito-core/javadoc.gradle

```diff
@@ -1,7 +1,31 @@
 //It seems the gradle javadoc task works file by file and as such disable some features of javadoc tool
 //such as link to packages, https://groups.google.com/d/msg/gradle-dev/R83dy_6PHMc/bgw0cUTMFAAJ
+
+def java8DocsUrl = 'https://docs.oracle.com/javase/8/docs/api/'
+def packageListUrl = "$java8DocsUrl/package-list"
+def java8DocsDir = "$buildDir/javase-8-docs"
+
+// Temporary workaround for https://github.com/mockito/mockito/issues/2765
+// For Java 8, download the `package-list` as `element-list`
+task downloadJava8ElementList(type: Download) {
+    src packageListUrl
+    // Since the JavaDoc output is generated in `$buildDir/javadoc`, specify a different location for this task
+    dest "$java8DocsDir/element-list"
+    overwrite true
+}
+
+// Temporary workaround for https://github.com/mockito/mockito/issues/2765
+// For Java 9+, download the `package-list` as-is
+task downloadJava9PlusPackageList(type: Download, dependsOn: downloadJava8ElementList) {
```

Comment:
Nit: rather than having the `dependsOn` here, let's add the `dependsOn` for both in the javadoc task. since the download of this task is independent of the previous task.